### DISCUSSION TITLE
Add power consumption

### DISCUSSION THREAD

**Discussion by tieko3000** [Rem0o/FanControl.Releases#2484] [Ideas]
Hello

 I love your work, thank you very much for your effort. 

Recently I have replaced the fan on my PSU and I am now controlling the new fan over my motherboard. 
Since there is no temperature reading for the PSU, I think it would be great to be able to add the power consumption of the CPU and GPU (in a mixed/accumulutated curve) as a reading to control the fan according to the load on the PSU.

Kind regards

**Comment by Rem0o** [comment]
You could simply create a mixed sensor which averages the temperature of your most power hungry components (CPU + GPU), and then feed that into a time average sensor.  Use that as a rough estimate of the load on the PSU.

**Comment by DragosValStefan** [comment]
I would also like power consumption as sensor source. It would enable more tight control over fan curves;
We could mix temp + Power and result in a more advanced way of reducing FAN speed maybe even passive cooling
